docs/dev/stream/operators/windows.md

```diff
@@ -111,6 +111,11 @@ windows) assign elements to windows based on time, which can either be processin
 time. Please take a look at our section on [event time]({{ site.baseurl }}/dev/event_time.html) to learn
 about the difference between processing time and event time and how timestamps and watermarks are generated.
 
+Time-based windows have a *start timestamp* (inclusive) and an *end timestamp* (exclusive)
+that together describe the size of the window. In code, Flink uses `TimeWindow` when working with
+time-based windows, this has methods for querying the start- and end-timestamp and also an
```

Comment:
"this has methods" => "which has methods"

Reply:
Fixed